Objects/Objects/GIS/GisTopography.cs

```diff
@@ -0,0 +1,22 @@
+using System.Collections.Generic;
+using Objects.BuiltElements.Revit;
+using Objects.Geometry;
+using Speckle.Core.Models;
+
+namespace Objects.GIS
+{
+  public class GisTopography : Base
```

Comment:
As suggested by Matteo, this may be better to inherit from `Topography`, in which case there will be no need for a specific conversion of GisTopography right now.